__tests__/extension-requests/extension-requests.test.js

```diff
@@ -874,4 +901,41 @@ describe('Tests the Extension Requests Screen', () => {
     logs = await extensionLogsForFirstER.$$('.log-div');
     expect(Array.from(logs).length).toBe(9);
   });
+
+  it('Should update page url when filters and usernames are changed', async () => {
+    await page.click('#filter-button');
+    await page.click('input[value="PENDING"]');
+    await page.click('input[value="APPROVED"]');
+    await page.click('#apply-filter-button');
+    await page.type('#assignee-search', 'sunny,randhir');
+    await page.keyboard.press('Enter');
+    await page.waitForNetworkIdle();
+    const url = page.url();
+    expect(url).toBe(
+      `${baseUrl}?order=asc&size=5&q=status%3AAPPROVED%2Cassignee%3Asunny%2Brandhir`,
```

Comment:
where is PENDING in this url since it got clicked in ln 907 before applying filter or am I missing something?

Reply:
Since PENDING is selected by default when the page loads, we are deselecting it. So we are not expecting PENDING in the url